lib/graph_ql/directive.rb

```diff
@@ -2,21 +2,21 @@ class GraphQL::Directive < GraphQL::ObjectType
   LOCATIONS = [
     ON_OPERATION =  :on_operation?,
     ON_FRAGMENT =   :on_fragment?,
-    ON_FIELD =      :on_field?,
+    ON_FIELD =      :on_field?
   ]
   LOCATIONS.each do |location|
-    define_method(location) { self.on.include?(location) }
```

Comment:
I like the explicit `self` here becuase it's a reminder that `on` comes from the object but `location` comes from the closure